hack/test-cmd.sh

```diff
@@ -142,7 +142,7 @@ os::cmd::expect_success_and_text "openshift ex validate master-config ${MASTER_C
 os::cmd::expect_success_and_text "openshift ex validate node-config ${NODE_CONFIG_DIR}/node-config.yaml" 'SUCCESS'
 # breaking the config fails the validation check
 cp ${MASTER_CONFIG_DIR}/master-config.yaml ${BASETMPDIR}/master-config-broken.yaml
-os::util::sed '7,12d' ${BASETMPDIR}/master-config-broken.yaml
+os::util::sed '12,17d' ${BASETMPDIR}/master-config-broken.yaml
```

Comment:
wouldn't a better way to break it be to append a duplicate key that points to a struct of the wrong type?
like `echo "
kubernetesMasterConfig: invalidData
" >> ${BASETMPDIR}/master-config-broken.yaml`